nav2_costmap_2d/include/nav2_costmap_2d/layer.hpp

```diff
@@ -123,18 +123,23 @@ class Layer  // TODO(mjeronimo): public nav2_util::LifecycleHelperInterface
   const std::vector<geometry_msgs::msg::Point> & getFootprint() const;
 
   /** @brief Convenience functions for declaring ROS parameters */
-  void declareParameter(const std::string & param_name, const rclcpp::ParameterValue & value);
-  bool hasParameter(const std::string & param_name);
+  void declareParameter(
+    const std::string & param_name,
+    const rclcpp::ParameterValue & value);
+  bool hasParameter(
+    const std::string & param_name);
   void undeclareAllParameters();
   std::string getFullName(const std::string & param_name);
 
 protected:
   LayeredCostmap * layered_costmap_;
   std::string name_;
   tf2_ros::Buffer * tf_;
-  nav2_util::LifecycleNode::SharedPtr node_;
   rclcpp::Node::SharedPtr client_node_;
```

Comment:
They were here before this PR, though I couldn't really find a proper use when I was going through the code. This could be refactored in a different PR.

Reply:
OK, can you file a ticket on this?